ncm-cron/src/test/resources/cron_syslog-common.pan

```diff
@@ -55,4 +55,11 @@ include 'components/cron/config';
     "command", "some command",
 ));
 
+"/software/components/cron/entries" = append(SELF, dict(
+    "name", "test_magic_auto_value",
+    "user", "magic",
+    "frequency", "AUTO 7 * * *",
```

Comment:
AUTO -> auto

Reply:
No, this is correct, existing code in template libraries only uses uppercase.
I've just updated the component to a case-insensitve substitution to allow either.

Reply:
but the scheam does not allow this

Reply:
I'll add test cases for both.